.github/knownwords.txt

```diff
@@ -1100,6 +1100,7 @@ certificateInformation
 changePasswordId
 changePasswordIdGenerator
 changePasswordIdTimeToLiveInSeconds
+completionTokenTimeToLiveInSeconds
```

Comment:
Would still like one of the other suggestions I made on the PR instead of 'completionToken'.

Reply:
Understood. I'm still waiting on another review for that one to get more feedback on the naming to avoid multiple changes. I decided to get PRs for the other repos prepped ahead of time. Then I can just run through for naming instead of delaying all the work until the app PR is reviewed.